app/src/main/java/app/revanced/manager/ui/component/ArrowButton.kt

```diff
@@ -26,4 +26,21 @@ fun ArrowButton(modifier: Modifier = Modifier, expanded: Boolean,onClick: () ->
                 .then(modifier)
         )
     }
+}
+
+@Composable
+fun Arrow(modifier: Modifier = Modifier, expanded: Boolean) {
+    val description = if (expanded) R.string.collapse_content else R.string.expand_content
+    val rotation by animateFloatAsState(
+        targetValue = if (expanded) 0f else 180f,
+        label = "rotation"
+    )
+
+    Icon(
+        imageVector = Icons.Filled.KeyboardArrowUp,
+        contentDescription = stringResource(description),
+        modifier = Modifier
+            .rotate(rotation)
+            .then(modifier)
+    )
```

Comment:
This does not seems to be used

Reply:
It's used in the steps and substeps, the IconButton added extra padding which i didnt want

Reply:
You are right. I didn't see it because github was hiding `PatcherScreen.kt`.